Makefile

```diff
@@ -49,7 +49,7 @@ test:
 .PHONY: test
 
 proto-gen:
-	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen:v0.2 sh ./scripts/protocgen.sh
+	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen:v0.7 sh ./scripts/protocgen.sh
```

Comment:
Wouldn't it make more sense if we use latest, instead of specifying a version? To know when we break compatibility again